app/javascript/services/forest-data.js

```diff
@@ -2,7 +2,9 @@ import axios from 'axios';
 
 const DATASET = process.env.COUNTRIES_PAGE_DATASET;
 const API_URL = process.env.GFW_API_HOST_PROD;
+const CARTO_API_URL = process.env.CARTO_API_URL;
 const REQUEST_URL = `${API_URL}/query/${DATASET}?sql=`;
+const CARTO_REQUEST_URL = `${CARTO_API_URL}/sql?q=`;
```

Comment:
Can we combine this with about also. As we are only declaring the carto url to use once.